src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
+  const res = sourceString.split(';')
+    .map(data => data.split(':').map(val => val.trim()));
```

Comment:
```suggestion
  const res = sourceString
    .split(';')
    .map(data => data
       .split(':')
       .map(val => val.trim())
     );
```
![image](https://user-images.githubusercontent.com/96409563/220746644-0362a43c-fd20-4ded-9b94-1d198b0a0ef2.png)